_gabarits-jsonld/README.md

```diff
@@ -47,9 +47,15 @@ Saisissez des types supplémentaires correspondant au type particulier de l’é
 ### [_name_](https://schema.org/name)
 Saisissez le titre de l’événement.
 
+### @id
+Saisissez une URL canonique pour cet événement, c'est-à-dire une URI (Uniform Resource Identifier) constituant un identifiant unique pour cet événement. L'URI de l'événement devrait idéalement être différente de l'URL de la page web de l'événement, puisqu'il s'agit de deux [_choses_](https://schema.org/Thing) (ou _entités_) distinctes dans le contexte des données structurées Schema.
+
+Une façon simple de générer une URI distincte pour chaque événement consiste à ajouter à la suite de l'URL de la page web une ancre ***#*** suivie d'une chaîne de caractère servant d'identifiant unique d'événement. Voici un exemple fictif : `https://diffuseurdespectacles.ca/evenements/nomduspectacle/#1324`. Dans cet exemple, l'URL de la page web est `https://diffuseurdespectacles.ca/evenements/nomduspectacle/`, alors que la chaîne de caractères `1324` représenterait l'identifiant unique de l'événement. N'importe quelle chaîne de caractères peut convenir du moment où elle constitue une ancre unique à l'intérieur de la page web de l'événement. Ce pourrait être un identifiant local propre au CMS du site web et désignant l'événement dans la base de données du système. Ce pourrait aussi être la date et l'heure de la représentation. Il n'est pas nécessaire qu'un point d'ancrage soit réellement présent dans le corps de la page web : du moment où le symbole dièse et une chaîne de caractères unique est ajoutée à l'URL de la page web, cela satisfait à l'exigence d'une URL canonique dans le contexte de Schema.org.
+
+Dans le cas de séries de représentations, en plus de l'URI  de la page web de la série, une URI distincte devrait être assignée à chaque représentation à l'aide de l'[Extension pour les Séries d'événements](https://github.com/culturecreates/artsdata-data-model/blob/master/_gabarits-jsonld/Event/event_series.jsonld). Si les différentes représentations ne disposent pas de leurs propres page web (en sus de la page de la série), il est tout de même possible de leur assigner des URI distinctes en ajoutant à l'URL de la page des ancres # suivies de la date et de l'heure de chaque représentation, tel qu'expliqué plus haut. Dans un tel scénario, assurez-vous que l'URI de l'objet de type _EventSeries_ soit disctincte des URIs des objets de type _SubEvent_.
```

Comment:
This is also tricky. Event series can have events on the same day at different times, so using the data without time can cause different events to have the same URI.